demo/full/mpd-parser.wasm

```diff
@@ -0,0 +1 @@
+../../dist/mpd-parser.wasm
```

Comment:
symbolic link to the mpd-parser built for the last version.

Though this means that we now may want to re-build it when it changes mid-version so that may not be what we want, should we instead built it at demo-building time or let the user take care of this?